### PR TITLE
ES Version revert to 7.0.0

### PR DESCRIPTION
**Pull request by robrankin**
As noted in https://github.com/Yelp/elastalert/issues/2725 more recent versions of elasticsearch pip package causes errors.  Started seeing these errors when moving to the newest release here.

Error:
```
deprecated_search() got an unexpected keyword argument 'headers'
```

I see this was updated in https://github.com/jertel/elastalert2/commit/45237cc364e2cafc1859d12841b1bab11c810d9f however from my experience it probably needs to be reverted for now.

~~Also a slight change to the Dockerfile.  If someones already cloned the repo, in order to work on the Dockerfile, having it RUN git clone again is confusing.~~

### CODE REVIEW

**Comment by nsano-rururu** [comment]
@robrankin

In the rule I created, there is no problem in the state of `elasticsearch>=7.0.0,<8.0.0`, but does the rule I create cause a problem?

**Comment by robrankin** [comment]
> In the rule I created, there is no problem in the state of `elasticsearch>=7.0.0,<8.0.0`, but does the rule I create cause a problem?

In my testing, the current version 7.12.0 causes the error, as mentioned.  Downgrading back to 7.0.0 stops the errors.

**Comment by nsano-rururu** [comment]
@robrankin

It's not a problem, but if there is a problem depending on the environment, I'm in favor of returning it.
As a reminder, does that mean that the current latest and latest elaticsearch library version of jertel/elastalert2, not yelp/elastalert, is a problem with the current settings and is no longer a problem with 7.0.0? ..

**Comment by robrankin** [comment]
> It's not a problem, but if there is a problem depending on the environment, I'm in favor of returning it.
> As a reminder, does that mean that the current latest and latest elaticsearch library version of jertel/elastalert2, not yelp/elastalert, is a problem with the current settings and is no longer a problem with 7.0.0? ..

Apologies, yes this problem is evident with elastalert2, specifically using the Docker container.

**Comment by nsano-rururu** [comment]
@robrankin

thank you for your answer.

**Comment by nsano-rururu** [comment]
doc_type If you continue to support what you specify,
It seems that it is necessary to fix it to elasticsearch == 7.
Looking at the past issues of elasticsearch-py, there was a comment that "It has been deleted in as many places as possible (or it is the default kwarg)", so the newer the version, the more it corresponds. Is being done a lot. Is it because doc_type is completely deleted in elasticsearch 8?

**Comment by jertel** [comment]
The Dockerfile needed to clone the repo since Docker Hub created the builds, and it doesn't have a local copy of the repo prior to running the docker build command.

I agree, though, that it is inefficient for us users. So I've disabled automated builds on Docker Hub and instead activated a new GitHub action to perform the image build and push, and removed the git clone from the Dockerfile. So this will resolve your concern, but you'll need to remove your Dockerfile changes from this PR to avoid merge conflicts with the changes I pushed this morning.

**Comment by robrankin** [comment]
Merge conflicts gone.

File: Dockerfile [hunk 1]
```diff
@@ -9,8 +9,12 @@ RUN apk --update upgrade && \
 
 COPY . /tmp/elastalert
 
-RUN mkdir -p /opt/elastalert && \
-    cd /tmp/elastalert && \
+
+RUN mkdir -p /opt/elastalert
+
+COPY . /tmp/elastalert
```
Comment (jertel) [inline Dockerfile:15]: This line duplicates line 10. I suggest reverting all your changes to Dockerfile since they are no longer needed.

**Review by jertel — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (robrankin) [reply Dockerfile]: All gone now.

**Review by jertel — APPROVED** [review]

**Comment by jertel** [comment]
I'll open an issue for the obsolete doc_type field. That is going to be a blocker when ES8 is released.

**Pull request merged**